res/img/edit.svg

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please ask @nadonomy to clean up this icon... It should be possible to be quite a bit smaller than this.